README.md

```diff
@@ -4,13 +4,60 @@ HasCore: HasGeek API
 ![Build status](https://secure.travis-ci.org/hasgeek/hascore.png)
 [![Coverage Status](https://coveralls.io/repos/hasgeek/hascore/badge.png?branch=master)](https://coveralls.io/r/hasgeek/hascore?branch=master)
 
-Hascore provides helper API methods including the networkbar that is shown
-on all sites, and API endpoints for geolocation and language analysis.
+Hascore provides helper API methods including the networkbar that is shown on all sites, and API endpoints for geolocation and language analysis.
 
-Hascore requires NLTK. You will need to install NLTK's corpus data.
-Be warned, this is about 1.8 GB:
+-----
+### NLTK
+
+Hascore requires NLTK. You will need to install NLTK's corpus data. Be warned, this is about 1.8 GB:
 
     $ python -m nltk.downloader all
+    
+### Postgres
+
+Hascore uses Postgres >=9.4 and Redis server for development. To set up a Postgres DB:
+
+On OS X using the [Postgres App](http://postgresapp.com):
```

Comment:
Oops this is my grammatical mistake and you copy pasted! It should read "In OS X.." and not "On", same for "One any Linux.."